wagtailio/project_styleguide/templates/patterns/pages/feature_index_page/feature_index_page.html

```diff
@@ -2,9 +2,30 @@
 {% load wagtailcore_tags wagtailimages_tags static %}
 
 {% block content %}
-    {% if page.subheading %}<h3>{{ page.subheading }}</h3>{% endif %}
-    {% for category in page.feature_categories %} ☐ {{ category }} {% endfor %}
-    {% include_block page.body %}
+
+    <div class="features" {% if page.feature_categories %}data-feature-index{% endif %}>
+        {% include "patterns/components/hero/hero.html" with value=page classes="hero--features" %}
+        <div class="grid">
+
+            {# Filters #}
+            <fieldset class="features__filters">
+                <div class="features__filter-wrap">
+                    <input type="checkbox" id="all" name="all" data-feature-filter-all checked>
+                    <label class="mini-meta" for="all">All</label>
+                </div>
+                {% for category in page.feature_categories %}
+                    <div class="features__filter-wrap">
+                        <input type="checkbox" id="{{ category|lower }}" name="{{ category|lower }}" data-feature-filter>
+                        <label class="mini-meta" for="{{ category|lower }}">{{ category }}</label>
+                    </div>
+                {% endfor %}
+            </fieldset>
+        </div>
+
+        {# Features #}
+        {% include_block page.body %}
+    </div>
+
     {% include_block page.cta %}
```

Comment:
Note: I think the spacing on this might need checking on the design it's full-width.

Reply:
Looks like it's using the wrong template. Should be [cta.html](https://wagtail-org-production.herokuapp.com/pattern-library/pattern/patterns/components/streamfields/cta/cta.html) rather than [cta_block.html](https://wagtail-org-production.herokuapp.com/pattern-library/pattern/patterns/components/streamfields/cta/cta_block.html). I'll make a note on the BE MR.